css/components/intro.css

```diff
@@ -1,32 +1,37 @@
 .intro {
-    padding: 20px;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    font-family: 'Roboto', sans-serif;
-
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-family: 'Roboto', sans-serif;
 }
 
 .intro__header {
-    text-transform: uppercase;
-    padding: 20px 0 10px 0;
+  text-transform: uppercase;
+  padding: 20px 0 10px 0;
 }
```

Comment:
Just a small hint: a better naming convention for this class could be `intro__heading` or `intro__title` because it can be a bit confusing regarding the html tag `header`.

Reply:
Thank you for your feedback! I will change the class-name.